src/_P035_IRTX.ino

```diff
@@ -123,8 +123,8 @@ boolean Plugin_035(byte function, struct EventStruct *event, String& string)
             printWebString += IrBLen;
             printWebString += F("<BR>");
 
-            uint16_t buf[200];
-            uint16_t idx = 0;
+            uint16_t buf[250];  //The Raw Timings that we can buffer.
```

Comment:
I think it is time to allocate these on the heap, since such  large buffers allocated on the stack will lead to issues which will be hard to reproduce.

Reply:
because i am a bit of a malloc noob will this do?

 ```
uint16_t *buf;
buf =  reinterpret_cast<uint16_t*>(malloc(250 * sizeof(uint16_t)));